docs/install/local/README.md

```diff
@@ -248,20 +259,26 @@ you can use a pre-built docker image that provides everything needed.
     This will map the `1883`/`1884` ports to the host machine so they can be accessed outside of the container. If you already have an MQTT broker running on port 1883, then you'll need to modify the `-p` options to use a different set of ports. For example: `-p 9883:1883 -p 9884:1884`.
 
 
-### Upgrade
+## Upgrade
 
 To upgrade to the latest release you can follow these steps. Replace `x.y.z` with the
 version you are upgrading to.
 
  1. Stop FlowForge `sudo service flowforge stop` [^2]
- 2. `cd /opt/flowforge/app`
- 3. `sudo -u flowforge npm install @flowforge/flowforge@x.y.z` [^3]
+ 2. Change into the `app` directory
+    * `cd /opt/flowforge/app` (Linux/MacOS)
+    * `cd c:\flowforge\app` (Windows)
+ 3. NPM install the desired version
+    * `sudo -u flowforge npm install @flowforge/flowforge@x.y.z` (Linux/MacOS) [^3]
+    * `npm install @flowforge/flowforge@x.y.z` (Windows)
  4. Check the release notes for any additional steps needed to upgrade the particular version
  5. Restart FlowForge `sudo service flowforge start` [^2]
  
 If you are running as your normal user you can drop the `sudo -u flowforge` and just run `npm install @flowforge/flowforge@x.y.z`
 
 ---
 
+
+[^1]: Arm6 devices, such as the original Raspberry Pi Zero and Zero W are not supported.
```

Comment:
Given these footnotes aren't rendered properly by our docs, we either need to add whatever markdown plugin is needed on the website, or stop using them and add these things as inline comments.

Reply:
Agree. I found the necessary Marked info for adding footnote support to Marked so what i did here was to group the footnotes at the footer in preparation.

I would like to keep this change and (eventually) the website will catch up.

Reply:
Issue raised here: https://github.com/flowforge/website/issues/250

Reply:
> I found the necessary Marked info for adding footnote support to Marked

Is that info shared in the issue or are you planning to PR the necessary changes?

Reply:
No, not yet Nick. Will update the issue next.

I can also self assign the issue if you wish?

Reply:
@Steve-Mcl if you know what needs doing, then please get it done.

Reply:
NP